src/main/java/com/celements/web/plugin/api/CelementsWebPluginApi.java

```diff
@@ -550,60 +541,6 @@ public XWikiUser checkAuth(String logincredential, String password, String remem
         possibleLogins, noRedirect);
   }
 
-  private PageLayoutCommand getPageLayoutCmd() {
-    if (!context.containsKey(CELEMENTS_PAGE_LAYOUT_COMMAND)) {
-      context.put(CELEMENTS_PAGE_LAYOUT_COMMAND, new PageLayoutCommand());
-    }
-    return (PageLayoutCommand) context.get(CELEMENTS_PAGE_LAYOUT_COMMAND);
-  }
-
-  /**
-   * @deprecated since 2.82 instead use
-   *             {@link LayoutScriptService #canRenderLayout(SpaceReference)}
-   */
-  @Deprecated
-  public boolean canRenderLayout(SpaceReference spaceRef) {
-    return getPageLayoutCmd().canRenderLayout(spaceRef);
-  }
-
-  /**
-   * @deprecated since 2.59 instead use
-   *             {@link LayoutScriptService #renderPageLayout(SpaceReference)}
-   */
-  @Deprecated
-  public String renderPageLayout(SpaceReference spaceRef) {
-    return getLayoutScriptService().renderPageLayout(spaceRef);
-  }
-
-  /**
-   * @deprecated since 2.59 instead use
-   *             {@link LayoutScriptService #renderCelementsDocumentWithLayout(DocumentReference,
-   *             SpaceReference)}
-   */
-  @Deprecated
-  public String renderCelementsDocumentWithLayout(DocumentReference docRef,
-      SpaceReference layoutSpaceRef) {
-    return getLayoutScriptService().renderCelementsDocumentWithLayout(docRef, layoutSpaceRef);
-  }
-
-  /**
-   * @deprecated since 2.18.0 instead use
-   *             {@link LayoutScriptService #renderPageLayout(SpaceReference)}
-   */
-  @Deprecated
-  public String renderPageLayout(String spaceName) {
-    return getPageLayoutCmd().renderPageLayoutLocal(getWebUtilsService().resolveSpaceReference(
-        spaceName));
-  }
-
-  /**
-   * @deprecated since 2.59 instead use {@link LayoutScriptService #renderPageLayout()}
-   */
-  @Deprecated
-  public String renderPageLayout() {
-    return getLayoutScriptService().renderPageLayout();
-  }
-
```

Comment:
still used in at least celements-photo

Reply:
https://github.com/celements/celements-image/pull/82